notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    if os.getenv("IS_TESTING"):
</pre><p>What is the reason we can't just use the 'else' branch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Reply:
In the testing environment, the model file stores in the local temp folder and uploads to the Cloud Storage bucket, while it uploads to the bucket directly in the Dataproc environment.

It's the same reason that we used `spark-bigquery-connector` from the github, not the Cloud Storage bucket. `BUCKET_URI` is starts with `gs://`, which requires additional steps only for the testing environment.